pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -279,7 +288,7 @@ var (
 	DefaultNodeSSHPublicKeyPath = "~/.ssh/id_rsa.pub"
 
 	// DefaultNodeVolumeType defines the default root volume type to use
-	DefaultNodeVolumeType = NodeVolumeTypeGP2
+	DefaultNodeVolumeType = NodeVolumeTypeGP3
```

Comment:
does changing the default effect the way we interactive with existing nodegrooups at all? I imagine this wasn't being explicitly set most if the time